### ISSUE TITLE
Unable to interactive with nested context menu

### ISSUE THREAD

**Issue by nblackburn** [TablePlus/TablePlus-Windows#493]
I am unable to interact with nested context menus with my mouse. I have to resort to using up/down on my keyboard in order to access critical functions.

## Steps to reproduce

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
PostgreSQL 13.3

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
186

3. The steps to reproduce this issue:
- Open TablePlus
- Connect to DB
- Right click left sidebar
- Try to access nested context menu

---

![2022-01-10_18-15-56](https://user-images.githubusercontent.com/2931085/148818014-fd00ec2b-794a-44b5-9d3c-cf8ff64f6274.gif)

**Comment by huyphams** [comment]
It is a really weird issue, we could not reproduce it.

**Comment by nblackburn** [comment]
@huyphams Yeah, it's made using the app infuriating and just started happening in a recent update.
